Branch-SDK/src/androidTest/java/io/branch/referral/BranchCPIDTest.java

```diff
@@ -32,8 +31,9 @@ public void testGetCPID() throws Throwable{
     @Test
     public void testGetLATD() throws Throwable{
         Branch branch = Branch.getInstance(getTestContext());
-
-        new BranchLastAttributedTouchData(null, getTestContext());
+        if (branch != null) {
```

Comment:
I am curious what made it necessary to do this check, and further -- how the test can pass if `branch == null`

Reply:
instinctual avoidance of NPE :) removed it.